public/angular/sass/app.scss

```diff
@@ -267,3 +272,22 @@ li.collapse, li.collapsing {
   height: 20px;
   background-image: url('../images/map-marker.png');
 }
+
+#wrapper {
+  min-height:100%;
+  position:relative;
+}
+
+footer {
+  position: absolute;
+  background-color: #009446;
+  height: 20px;
+  width: 100%;
+  bottom: 0px;
+  left: 0px;
+  .version {
+    margin-top: 5px;
+    position: absolute;
+    right: 10px;
```

Comment:
Can the bg of the footer be a light gray color instead of the heavy green. Also can we get a bit more vertical padding both within the footer, and between the main content and the footer. You can see here how it's tight:

![image](https://cloud.githubusercontent.com/assets/702320/5722460/419ecb82-9b0a-11e4-9864-36df0ce71465.png)